custom/abt/reports/data_sources/late_pmt.json

```diff
@@ -32,16 +32,34 @@
     "engine_id": "default",
     "base_item_expression": {},
     "configured_filter": {
-      "expression": {
-          "type": "property_path",
-          "property_path": [
-            "user_data",
-            "usertype"
-          ]
-      },
-      "operator": "eq",
-      "property_value": "pmt",
-      "type": "boolean_expression"
+      "type": "and",
+      "filters": [
+        {
+          "expression": {
+            "type": "property_path",
+            "property_path": [
+              "user_data",
+              "usertype"
+            ],
+            "datatype": null
+          },
+          "operator": "eq",
+          "property_value": "pmt",
+          "type": "boolean_expression",
+          "comment": null
+        },
+        {
+          "expression": {
+            "type": "property_name",
+            "property_name": "is_active",
+            "datatype": null
+          },
+          "operator": "eq",
+          "property_value": "True",
```

Comment:
I think this needs to be
```JSON
"property_value": true, 
```
not sure, though - were you able to test it?

Reply:
No, not really, as I was told by @karmuno:

> In this case, the Late PMT (static) report is configured to pull directly from the CommCare User database, represented in the JSON in line 162: https://github.com/dimagi/commcare-hq/blob/3cda64f2f492ee9be4ff185625aa1394ef5054c8/custom/abt/reports/data_sources/late_pmt.json#L162 Our UCR GUI only has options for case and form datasources, which is why the "Source Type" shows up as "case" on HQ, even though it's not. Because of this, you actually can't configure this type of datasource through the web interface. Unfortunately, it must be checked in through GitHub and then tested.

Should I change `"True"` to `true`?

Reply:
Yes, I just tested it out, it does need to be `true`, not `"True"`.

@karmuno is correct in that this report can't be edited in the UI, but if you have a functional local development environment, you should be able to test it there.

Reply:
Ok, thanks for the information.